plugins/inputs/icinga2/README.md

```diff
@@ -0,0 +1,62 @@
+# Example Input Plugin
+
+This plugin gather services & hosts status using Icinga2 Remote API.
+
+The icinga2 plugin uses the icinga2 remote API to gather status on running
+services and hosts. You can read Icinga2's documentation for their remote API
+[here](https://docs.icinga.com/icinga2/latest/doc/module/icinga2/chapter/icinga2-api)
+
+### Configuration:
+
+```toml
+# Description
+[[inputs.icinga2]]
+  ## Required Icinga2 server address (default: "https://localhost:5665")
+  # server = "https://localhost:5665"
+  
+  ## Required Icinga2 object type ("services" or "hosts, default "services")
+  # object_type = "services"
+
+  ## Credentials for basic HTTP authentication
+  # username = "admin"
+  # password = "admin"
+
+  ## Maximum time to receive response.
+  # response_timeout = "5s"
+
+  ## Optional TLS Config
+  # tls_ca = "/etc/telegraf/ca.pem"
+  # tls_cert = "/etc/telegraf/cert.pem"
+  # tls_key = "/etc/telegraf/key.pem"
+  ## Use TLS but skip chain & host verification
+  # insecure_skip_verify = true
+```
+
+### Measurements & Fields:
+
+- All measurements have the following fields:
+    - name (string)
+    - state (int)
+
+### Tags:
+
+- All measurements have the following tags:
+    - check_command
+    - display_name
+    - source
```

Comment:
Although not documented yet, as discussed in https://github.com/influxdata/telegraf/issues/4413 we are planning to standardize on `source` as the hostname of the metric source and split out other components.  Can you change the style of these tags to be:
```
source=example.org,port=5665,scheme=https
```

Reply:
done